module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,69 @@
+#include "compression.hpp"
+
+#include <iostream>
+
+void printMap(const std::array<std::array<uint8_t, height>, width>& input) {   
+    for(size_t i = 0; i < height; ++i) {
+        for(size_t j = 0; j < width; j++) {
+            std::cout << input[i][j] << ' ';
+        }
+        std::cout << '\n';
+    }
+    std::cout << '\n';
+}
```

Comment:
What about non-printable characters?
You should print only one character per pixel, don't add a space.